packages/orchestration/src/utils/zoe-tools.js

```diff
@@ -114,21 +115,24 @@ export const makeZoeTools = (
         throw makeError(`One or more deposits failed ${q(errors)}`);
       }
       // TODO #9541 remove userSeat from baggage
-    },
-  );
+    });
 
   /**
    * Transfer the `amounts` from a `localAccount` to the `recipientSeat`. If any
    * of the withdrawals fail, everything will be rolled back to the
    * `srcLocalAccount`. Supports multiple items in the `amounts`
-   * {@link PaymentKeywordRecord}.
+   * {@link PaymentKeywordRecord}
+   *
+   * @type {HostOf<WithdrawToSeat>}
    */
-  const withdrawToSeat = retriable(
-    zone,
-    'withdrawToSeat',
-    /** @type {WithdrawToSeat} */
-    async (localAccount, destSeat, amounts) => {
-      await null;
+  const withdrawToSeat = (localAccount, destSeat, amounts) =>
```

Comment:
is it possible for a vat upgrade/restart to cause one of these promises to reject before the vow is complete?

Reply:
got an answer from @mhofman : a promise resolved promptly is currently safe from being severed by an upgrade because we only trigger vat upgrades as the result of network input